virt/virtual_machines/vm_networking/virt-attaching-vm-to-external-network.adoc

```diff
@@ -0,0 +1,80 @@
+
+Depending on the type of networks you are using, it may be necessary to update your xref:../../../virt/node_network/virt-updating-node-network-config.adoc#virt-about-nmstate_virt-updating-node-network-config[node networking configuration].
+
+Once a network bridge is created, you can start virtual machines on the external network in the same manner as you start virtual machines attached to internal networks, by using PXE booting, for example.
+
+= About configuring network bridges
+
+To attach virtual machines to additional networks, you must configure a Linux bridge.
+
+To attach a virtual machine to an additional network:
+
+. Configure a Linux network bridge.
+. Configure a bridge network attachment definition for a namespace in the web console or CLI.
++
+[NOTE]
+====
+The network attachment definition must be in the same namespace as the pod or virtual machine.
+====
+. Attach the virtual machine to the network attachment definition by using either the web console or the CLI:
+** In the web console, create a NIC for a new or existing virtual machine.
+** In the CLI, include the network information in the virtual machine configuration.
+
+[NOTE]
+====
+There are multiple methods for configuring a VLAN, including network attachment definition and node network configuration policy.
+However, a network attachment definition provides a more efficient and more manageable configuration.
+====
+
+[id="virt-configuring-linux-bridge"]
+= Configuring a Linux bridge
+
+As a network administrator, you can create a Linux bridge interface on nodes in the cluster by applying a `NodeNetworkConfigurationPolicy` manifest to the cluster.
+
+include::modules/virt-creating-linux-bridge-nncp.adoc[leveloffset=+1]
+
+For more information about scheduling, interface types, and other node networking activities,
+see the xref:../../../virt/node_network/virt-updating-node-network-config.adoc#virt-about-nmstate_virt-updating-node-network-config[node networking]
+section.
+
+[id="virt-creating-network-attachment-definition"]
+= Creating a network attachment definition
+
+[WARNING]
+====
+Configuring ipam in a network attachment definition for virtual machines is not supported.
+====
+
+include::modules/virt-creating-bridge-nad-web.adoc[leveloffset=+2]
+
+include::modules/virt-creating-bridge-nad-cli.adoc[leveloffset=+2]
+
+[id="virt-attaching-vm-to-additional-network"]
+= Configuring the virtual machine to the additional network
+
+include::modules/virt-vm-creating-nic-web.adoc[leveloffset=+2]
+
+include::modules/virt-networking-wizard-fields-web.adoc[leveloffset=+2]
+
+include::modules/virt-attaching-vm-secondary-network-cli.adoc[leveloffset=+2]
+
+include::modules/virt-networking-glossary.adoc[leveloffset=+1]
+
+[id="{context}_additional-resources"]
+[role="_additional-resources"]
+== Additional resources
```

Comment:
I recommend removing this section. The links point to things you can do with a Linux bridge **after** it is created. They do not contain info that the user would need to create or configure the bridge.

Reply:
Removed. I do think these links are beneficial to the user as references, but I also know we can go overboard with adding these types of post-task links and then the whole topic resembles a dump that is hard to maintain.... so, your call.